envcov.sh

```diff
@@ -33,7 +33,7 @@ then
 fi
 
 LCOV_PATH="${XCODECOVERAGE_PATH}/lcov-1.11/bin"
-OBJ_DIR=${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}
+OBJ_DIR="${OBJECT_FILE_DIR_normal}/${CURRENT_ARCH}"
 
 # Fix for the new LLVM-COV that requires gcov to have a -v paramter
 LCOV() {
```

Comment:
Actually, looking at line 40 - `XCODECOVERAGE_PATH` and `LCOV_PATH` are both quoted when generated, so we shouldn't need quotes around either of those for this to work.